src/services/completions.ts

```diff
@@ -965,6 +965,9 @@ namespace ts.Completions {
                     symbolToOriginInfoMap[getSymbolId(firstAccessibleSymbol)] =
                         !moduleSymbol || !isExternalModuleSymbol(moduleSymbol) ? { kind: SymbolOriginInfoKind.SymbolMemberNoExport } : { kind: SymbolOriginInfoKind.SymbolMemberExport, moduleSymbol, isDefaultExport: false };
                 }
+                else {
+                    symbols.push(symbol);

```

Comment:
Seems like you could just have a `symbols.push(symbol)` at the end and use an early return in the inner-most `if`

Reply:
@DanielRosenwasser done.